### PR TITLE
Omnipod Dash support

### PR DESCRIPTION
**Pull request by gniezen**
This PR adds support for Omnipod Dash and updates to Electron v3 (as we need Node v10 for the libmtp wrapper I wrote). It also updates all the dependencies that do no require changes.

### CODE REVIEW

**Comment by gniezen** [comment]
[New Omnipod](https://trello.com/c/M1wDdLw5/452-new-omnipod)

File: lib/drivers/insulet/insuletDriver.js [hunk 1]
```diff
@@ -205,7 +216,7 @@ module.exports = function (config) {
       'hours'
     ]
     },
-    0x0004: { value: 0x0004, name: 'Bolus', format: 'isss', fields: [
+    0x0004: { value: 0x0004, name: 'Bolus', format: 'ishs', fields: [
```
[the remark this answers is not in the document]
Comment (gniezen) [reply lib/drivers/insulet/insuletDriver.js]: I was wondering the same thing, so on Friday I tested using a t:slim G4 and it works fine. It's basically just Tandem "unreserving" a field that was previously marked as "Reserved".

[the remark this answers is not in the document]
Comment (pazaan) [reply lib/drivers/insulet/insuletDriver.js]: Makes perfect sense 😄

**Comment by gniezen** [comment]
@pazaan Please review changes since last review.

**Comment by gniezen** [comment]
That is a great idea, and I should've thought of that before starting to implement DASH support. The release of DASH support is getting time-sensitive, so I'm not sure we should delay it any further. Updating the Omnipod driver to ES6 is not going to be a quick task.

**Comment by gniezen** [comment]
OK, so we've had an issue where the `node-mtp` module would just be missing from the final packaged app. After much trial and error, it turns out that `libusb` needs to be installed on the CI machine for the `node-mtp` module to make its way into the final app. 

If there's not an easy way to see the differences between what was previously reviewed and what changes are new (without looking at all the individual commits) this may help:
https://github.com/tidepool-org/chrome-uploader/compare/4d0019bb4ca4d4bb2ee2792940c2bc79157e35d8...omnipod-dash

**Pull request merged**